zap/src/main/java/org/zaproxy/zap/extension/ascan/VariantFactory.java

```diff
@@ -135,7 +135,9 @@ public List<Variant> createVariants(ScannerParam scanOptions, HttpMessage messag
         }
 
         if ((targets & ScannerParam.TARGET_COOKIE) != 0) {
-            listVariant.add(new VariantCookie());
+            VariantCookie vc = new VariantCookie();
+            vc.setEncodeCookieValues(scanOptions.isEncodeCookieValues());
```

Comment:
Would it be "better" to have a constructor that accepts a boolean and then have the existing constructor deprecated and use "false"? Then setup of the Variant isn't a two step process?

Reply:
I thought of that, but ended up copying the way VariantJSONQuery did it above. 
Wasnt sure what the implications would be, but a deprecated constructor would solve that..